Tasks/SonarQubePreBuild/SonarQubePreBuild.ps1

```diff
@@ -13,51 +13,34 @@ param(
 
 Write-Verbose "Starting SonarQube Pre-Build Setup Step"
 
-Write-Verbose -Verbose "connectedServiceName = $connectedServiceName"
-Write-Verbose -Verbose "projectKey = $projectKey"
```

Comment:
Just tried this out and that is indeed the case.
Write-Verbose -Verbose : Always prints the message
Write-Verbose : Prints message only if system.debug build variable is set to true